README.md

```diff
@@ -35,9 +35,13 @@ Prior to vSphere 8 Update 1, the Supervisor Services are only available with Sup
 
 3. Select `My Downloads` on the left hand side navigation.
 
-4. Select `Free Software Downloads available HERE`, if you are looking for free services. Search for `vSphere Supervisor Services`.
+4. Select `Free Software Downloads available HERE`, if you are looking for free services. **Use this option when you have a VVF or legacy entitlements**. Search for `vSphere Supervisor Services`
```

Comment:
I think the "if you are looking for free services." is confusing.
We should maybe say "if you are looking for free services OR you have VVF/legacy entitlements". ?